index.bs

```diff
@@ -426,12 +424,16 @@ To <dfn>parse an attribution destination</dfn> from a string |str|:
 
 <h3 algorithm id="noising-attribution">Noising attribution</h3>
 
-To <dfn>select a random attribution mode</dfn> given a double |noiseRate|, run the following steps:
+To <dfn>select a random attribution mode</dfn> given doubles |noiseRate| and |falselyRate|, run the
+following steps:
 
 1. Assert: |noiseRate| is between 0 and 1 (both inclusive).
+1. Assert: |falselyRate| is between 0 and 1 (both inclusive).
 1. Let |r| be a random double between 0 (inclusive) and 1 (exclusive).
 1. If |r| is greater than or equal to |noiseRate|, return "<code>truthfully</code>".
-1. Return "<code>never</code>".
+1. Let |t| be a random double between 0 (inclusive) and 1 (exclusive).
```

Comment:
I don't think this will work as expected (let me know if my understanding is wrong).

The choice of falsely is conditioned on 1 - noiseRate, and so it will happen with (1-noise rate) * falsely rate probability.

Reply:
Is that different from [what the implementation does now](https://source.chromium.org/chromium/chromium/src/+/master:content/browser/attribution_reporting/attribution_policy.cc;l=121-141;drc=a67b4875d7d7f06ebbd559b591198c3eec181968)?

Reply:
Ah, I misunderstood the meaning of falselyRate, this does work fine.

I think it might be a bit more intuitive to change the approach here to specify two rate:

% of sources which get never
% of sources which get falsely

If a UA wanted to vary one of the rate, it would require them to do a bit more math to get the constants. wdyt?